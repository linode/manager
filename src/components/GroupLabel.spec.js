describe('components/GroupLabel', () => {
  it.skip('should render without error', () => {
  });
});
