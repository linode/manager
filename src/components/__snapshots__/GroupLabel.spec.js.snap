// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/GroupLabel should render without error 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <GroupLabel
    object={
        Object {
            "group": "A",
            "label": "B",
          }
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <span
          className="text-muted"
>
          A
           / 
</span>,
        <strong>
          B
</strong>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "A",
            " / ",
          ],
          "className": "text-muted",
        },
        "ref": null,
        "rendered": Array [
          "A",
          " / ",
        ],
        "type": "span",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "B",
        },
        "ref": null,
        "rendered": "B",
        "type": "strong",
      },
    ],
    "type": "span",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <span
            className="text-muted"
>
            A
             / 
</span>,
          <strong>
            B
</strong>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "A",
              " / ",
            ],
            "className": "text-muted",
          },
          "ref": null,
          "rendered": Array [
            "A",
            " / ",
          ],
          "type": "span",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "B",
          },
          "ref": null,
          "rendered": "B",
          "type": "strong",
        },
      ],
      "type": "span",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactFifteenAdapter {
      "options": Object {
        "supportPrevContextArgumentOfComponentDidUpdate": true,
      },
    },
  },
}
`;
